packages/accordions/src/styled/accordion/StyledRotateIcon.ts

```diff
@@ -0,0 +1,44 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import { cloneElement, Children } from 'react';
+import styled from 'styled-components';
+import { getColor, retrieveComponentStyles, DEFAULT_THEME } from '@zendeskgarden/react-theming';
+
+const COMPONENT_ID = 'accordions.rotate_icon';
+
+export interface IStyledRotateIcon {
+  isExpanded?: boolean;
+  isCompact?: boolean;
+}
+
+/* eslint-disable-next-line @typescript-eslint/no-unused-vars */
+export const StyledRotateIcon = styled(({ children, isRotated, isHovered, isCompact, ...props }) =>
+  cloneElement(Children.only(children), props)
+).attrs({
+  'data-garden-id': COMPONENT_ID,
+  'data-garden-version': PACKAGE_VERSION
+})<IStyledRotateIcon>`
+  transform: ${props => props.isRotated && `rotate(${props.theme.rtl ? '-' : '+'}180deg)`};
+  transition: transform 0.25s ease-in-out;
```

Comment:
Should we add a min width/height designation here of some sort to account for the dropdown chevron getting smaller as it's scaled down? Doesn't seem to happen unless more elements are added inside the `Header`: https://share.getcloudapp.com/xQuWWPd8

Reply:
@rossmoody @hzhu not exactly sure what is happening here. designs did not intend for the chevron to change in size.

Reply:
Yeah the element needs `min-width` / `min-height` 👍 .